README.md

```diff
@@ -1,34 +1,30 @@
 # RHTAP Observability 
 
-This repository contains Prometheus recording rule files and alert rule files for
-observability use, as well as unit tests for these rules.
+This repository is maintained by the RHTAP O11Y team.
 
-## Recording Rules
+The repository contains Prometheus alert rules [files](rhobs/alerting) for monitoring
+RHTAP data plane clusters along with their [tests](test/promql).
 
-- `appstudio_container_network_transmit_bytes_total` collects the network egress of
-containers and adds the `label_pipelines_appstudio_openshift_io_type` label to our
-metric.
+Control plane clusters alert rules are maintained as well by the same team, but are
```

Comment:
Current: "Control plane clusters alert rules are maintained as well by the same team"

Suggestion: "Control plane clusters alert rules are maintained by the o11y team as well"

Reply:
changed